README.md

```diff
@@ -1,14 +1,104 @@
-# Project
+# Microsoft Entra Wallet Library
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+## Introduction
+The Microsoft Entra Wallet Library for iOS gives your app the ability to begin using the Microsoft Identity platform by supporting the issuance and presentation of Verified Ids in accordance with OpenID Connect, Presentation Exchange, Verifiable Credentials, and more up and coming industry standards.
 
-As the maintainer of this project, please make a few updates:
+---
+## Installation
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+You can use cocoapods to install the Wallet Library by adding it to your Podfile:
+```ruby
+
+target "YourApp" do
+  use_frameworks!
+  pod "WalletLibrary", "~> 0.0.1"
+end
+```
+> note: use_frameworks! is required for this Pod.
+---
+## Quick Start
+
+Here is a simple example of how to use the library. For more in-depth examples, check out the sample app.
+ 
+```Swift
+
+/// Create a verifiedIdClient.
+let verifiedIdClient = VerifiedIdClientBuilder().build()
+
+/// Create a VerifiedIdRequestInput using a OpenId Request Uri.
+let input = VerifiedIdRequestURL(url: URL(string: "openid-vc...")!)
```

Comment:
Perhaps include `://` as part of the string before truncating. As is, the string seems ambiguous on if its a schema or hostname.